Source/WebCore/Modules/modern-media-controls/controls/icon-service.js

```diff
@@ -35,6 +35,7 @@ const Icons = {
     InvalidCircle   : { name: "InvalidCircle", type: "pdf", label: UIString("Invalid") },
     InvalidPlacard  : { name: "invalid-placard", type: "png", label: UIString("Invalid") },
     Overflow        : { name: "Overflow", type: "svg", label: UIString("More\u2026") },
+    OverflowPlain   : { name: "OverflowPlain", type: "svg", label: UIString("More\u2026") },
```

Comment:
NIT: Maybe instead of calling this "plain" we call it `OverflowDots` since that's really what it's about (and also on macOS the overflow icon is ">>", so it's a bit odd to jump from ">>" to "..." by calling it "plain" 😅)